sdk/highlight-run/package.json

```diff
@@ -89,14 +89,14 @@
 		"@graphql-codegen/typescript-graphql-request": "^6.0.1",
 		"@graphql-codegen/typescript-operations": "^4.0.1",
 		"@opentelemetry/api": "^1.9.0",
-		"@opentelemetry/exporter-metrics-otlp-http": ">=0.57.1",
```

Comment:
Updating these version constraints seems to have solved the de-dupe problem. 
`sdk-trace-web` requires `1.30.11` of `sdk-trace-base`, but when these install a `0.200` version yarn just loses its mind and uses `sdk-trace-base` 2.x.